pkg/doc/verifiable/credential_test.go

```diff
@@ -78,8 +78,8 @@ func TestNewCredential(t *testing.T) {
 		// validate not null credential subject
 		require.NotNil(t, vc.Issuer)
 		require.Equal(t, "did:example:76e12ec712ebc6f1c221ebfeb1f", vc.Issuer.ID)
-		require.Equal(t, "Example University", vc.Issuer.Name)
-		require.Equal(t, "data:image/png;base64,iVBOR", vc.Issuer.Image)
+		require.Equal(t, "Example University", vc.Issuer.CustomFields["name"])
+		require.Equal(t, "data:image/png;base64,iVBOR", vc.Issuer.CustomFields["image"])
```

Comment:
do we have a test somewhere to ensure that custom fields are marshalled when outputting JSON?

Reply:
Yes, I added a new test called `TestMarshalIssuer()` for this purpose.